packages/main/src/plugin/index.ts

```diff
@@ -1447,9 +1447,12 @@ export class PluginSystem {
         taskId?: number,
         target?: string,
       ): Promise<unknown> => {
+        const targetDisplay = target ? ` (${target})` : '';
+
         // create task
         const task = taskManager.createTask({
-          title: `Building image ${imageName ?? ''}`,
+          title: `Building image ${imageName ?? ''}${targetDisplay}`,
```

Comment:
as already said, it's missing a unit test

suggestion: in your case, to handle nicely spaces/extra leading space/condition etc might be easier to push to an array and use join(' ') at the end

```ts
array = ['Building image']
if (imageName) {
  array.push(imageName)
}
if (target) {
  array.push(...)
}

title: array.join(' '),
```